agent/lib_monolog.c

```diff
@@ -27,6 +30,11 @@
 #define LOG_DECORATE_PROC_FUNC_NAME \
   "newrelic_phpagent_monolog_decorating_processor"
 
+/*
+ * Prefix for log context attributes 
+ */
+#define NR_LOG_CONTEXT_DATA_ATTRIBUTE_PREFIX "context."
```

Comment:
It looks like this macro is only used once [here](https://github.com/newrelic/newrelic-php-agent/pull/749/files#diff-d48bf43b4158fad631ce65b0464841310be10cff57ac0b937061fb9d6c33bdc3R459). Do you think it will be reused again in the future? If not, I would consider declaring it as a `const char *` instead directly in that function so the value of the variable is visible without having to scroll to the top of the file.

Reply:
I would prefer keeping it as a define and not hard code it into a local variable.  I could not rule out its use in other parts of the code at this point.